src/sentry/api/endpoints/organization_details.py

```diff
@@ -168,6 +168,8 @@ class OrganizationSerializer(BaseOrganizationSerializer):
     allowJoinRequests = serializers.BooleanField(required=False)
     relayPiiConfig = serializers.CharField(required=False, allow_blank=True, allow_null=True)
     apdexThreshold = serializers.IntegerField(min_value=1, required=False)
+    providerName = serializers.CharField(required=False, allow_blank=True, allow_null=True)
```

Comment:
Might be better to rename as `providerKey`. 

Name = It might be arbitrary from the user.

Key = There's usually a list of enums somewhere.

Reply:
Changed here and in tests to `providerKey`